doc/api/stream.md

```diff
@@ -531,8 +531,7 @@ added: v9.3.0
 
 * {number}
 
-Return the value of `highWaterMark` passed when constructing this
-`Writable`.
+Return the value of `highWaterMark` passed when creating this `Writable`.
```

Comment:
Just curious that why do we change the `constructing` to `creating`?

Reply:
https://github.com/nodejs/node/pull/29656#issuecomment-538737766